recipes/espei/meta.yaml

```diff
@@ -0,0 +1,68 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "espei" %}
+{% set version = "0.3" %}
+{% set sha256 = "cef90367df857468893c6c0949c5e5b4008815bdd8e20be612e94ec82964344d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
It looks like this package has a python entry_point:

https://github.com/PhasesResearchLab/ESPEI/blob/master/setup.py#L62-L63

Please add it to the `build` section:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points